socialhome/streams/app/components/stamped_elements/ProfileStampedElement.vue

```diff
@@ -72,23 +72,23 @@
                     <i class="fa fa-users" />
                     <i class="fa fa-arrow-right" />
                     <i class="fa fa-user" />
-                    &nbsp;{{ profile.followersCount }}
+                    &nbsp;{{ profile.followers_count }}
                 </b-button>
             </div>
         </div>
         <div class="d-inline-block">
             <img
-                v-if="profile.saferImageUrlLarge"
+                v-if="profile.image_url_medium"
```

Comment:
Why switch to medium? Large is 300px which is what is currently in the profile streams. Medium is 100px which is too small imho.